examples/01_plotting/plot_demo_glass_brain_extensive.py

```diff
@@ -102,6 +102,7 @@
     threshold=2,
     symmetric_cbar=False,
     cmap="viridis",
+    title="only plot positive values",
```

Comment:
those plots were added in the #3993 
just adding some title so the figures are easier to identify when running the example dumps > 20 figures on you